lib/cli/src/generators/REACT_NATIVE/index.ts

```diff
@@ -1,9 +1,9 @@
-import shell from 'shelljs';
 import chalk from 'chalk';
-import { paddedLog, copyTemplate } from '../../helpers';
-import { NpmOptions } from '../../NpmOptions';
-import { baseGenerator, GeneratorOptions } from '../baseGenerator';
+import shell from 'shelljs';
+import { getBabelDependencies, paddedLog, copyTemplate } from '../../helpers';
 import { JsPackageManager } from '../../js-package-manager';
+import { NpmOptions } from '../../NpmOptions';
+import { GeneratorOptions } from '../baseGenerator';
```

Comment:
is `baseGenerator` still used anywhere?

Reply:
GeneratorOptions is the type for the options parameter on the generator function